components/TextFields/src/ColorThemer/MDCTextFieldColorThemer.h

```diff
@@ -16,9 +16,11 @@
 
 #import <Foundation/Foundation.h>
 
-@protocol MDCColorScheme;
+@protocol MDCColorScheming;
```

Comment:
This should just import the `MaterialColorScheme.h` header. Generally-speaking we avoid forward declaring symbols from components that aren't in the current target (e.g. we wouldn't forward declare UIView or UITableViewDelegate).

Reply:
Ok. Is this captured somewhere, I've been adding forward declarations as much as possible recently.

Reply:
Done.